README.md

```diff
@@ -1,6 +1,8 @@
 # react-carousel
 
-> Feature-rich, react-way carousel component that does not suck 
+> Feature-rich, react-way carousel component that does not suck
+
+![gif](./assets/carousel-gif.gif)
```

Comment:
IMO it would be greak if the gif would be a link to docs/ demo like [here](https://github.com/juliangarnier/anime)

Reply:
added